### ISSUE TITLE
[sonarqube] Issue with sonarqube-sonarqube-install-plugins configMap

### ISSUE THREAD

**Issue by mrotrebor** [Oteemo/charts#208]
Hi, 

It seems there is an issue with sonarqube-sonarqube-install-plugins configMap creation when you install plugins through values.yaml

The problem is that it's needed to remove the first line of install_plugin.sh 
`rm /opt/sonarqube/extensions/downloads/*` 
but also it's needed reformat the script when you want to install more than one plugin. 

Here is part of my values.yaml
```
image:
  repository: sonarqube
  tag: 8.4.2-community

plugins:
  install:
    - "https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.5.0/sonarqube-community-branch-plugin-1.5.0.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-java-plugin/sonar-java-plugin-6.9.0.23563.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-css-plugin/sonar-css-plugin-1.3.1.1642.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-javascript-plugin/sonar-javascript-plugin-6.6.0.13923.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-typescript-plugin/sonar-typescript-plugin-2.1.0.4359.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-xml-plugin/sonar-xml-plugin-2.0.1.2020.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-jacoco-plugin/sonar-jacoco-plugin-1.1.0.898.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-html-plugin/sonar-html-plugin-3.2.0.2082.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-scm-git-plugin/sonar-scm-git-plugin-1.12.1.2064.jar"
    - "https://binaries.sonarsource.com/Distribution/sonar-go-plugin/sonar-go-plugin-1.8.0.1775.jar"
    #- "https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.0.1/sonar-findbugs-plugin-4.0.1.jar" # There is a bug in spotBugs 4.1.2. Waiting version with spotBugs 4.1.4 where the bug is fixed.
  lib:
    - sonarqube-community-branch-plugin-1.5.0.jar
```

And here is the configMap created: 

```
apiVersion: v1
data:
  install_plugins.sh: |-
    rm /opt/sonarqube/extensions/downloads/*
    echo "https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.5.0/sonarqube-community-branch-plugin-1.5.0.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-java-plugin/sonar-java-plugin-6.9.0.23563.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-css-plugin/sonar-css-plugin-1.3.1.1642.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-javascript-plugin/sonar-javascript-plugin-6.6.0.13923.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-typescript-plugin/sonar-typescript-plugin-2.1.0.4359.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-xml-plugin/sonar-xml-plugin-2.0.1.2020.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-jacoco-plugin/sonar-jacoco-plugin-1.1.0.898.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-html-plugin/sonar-html-plugin-3.2.0.2082.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-scm-git-plugin/sonar-scm-git-plugin-1.12.1.2064.jar" >> /opt/sonarqube/extensions/downloads/listecho "https://binaries.sonarsource.com/Distribution/sonar-go-plugin/sonar-go-plugin-1.8.0.1775.jar" >> /opt/sonarqube/extensions/downloads/list
    cat /opt/sonarqube/extensions/downloads/list | xargs -n 1 -P 8 wget --directory-prefix /opt/sonarqube/extensions/downloads --no-verbose
    rm /opt/sonarqube/extensions/downloads/list
    cp -f /opt/sonarqube/extensions/downloads/sonarqube-community-branch-plugin-1.5.0.jar /opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.5.0.jar
kind: ConfigMap
metadata:
  creationTimestamp: "2020-11-03T19:08:02Z"
  labels:
    app: sonarqube
    chart: sonarqube-9.1.1
    heritage: Helm
    release: sonar
  name: sonar-sonarqube-install-plugins
  namespace: sonar
```
There is a missing new line after every plugin that provoke 'listecho', it seems.

**Comment by mrotrebor** [comment]
I've created this PR: https://github.com/Oteemo/charts/pull/209

**Comment by mrotrebor** [comment]
The problem is fixed on version 9.1.2 of the chart.
